tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -81,12 +90,17 @@ Verify Automatically Detects a Group Selected Is Removed and Notify the User
 *** Keywords ***
 Teardown Admin UI
     [Documentation]  Setup Default Values In User Management Settings
+    Revert Changes To Access Configuration
+    #RHOSi Teardown
```

Comment:
Is rhosi teardown not needed anymore here?

Reply:
restored